src/yang/example-compa-diffserv.yang

```diff
@@ -32,60 +32,76 @@ module example-compa-diffserv {
      companyA diffserv specification extension.";
   revision YYYY-MM-DD {
     description
-      "Initial revision for diffserv actions on network packets";
+      "Initial revision for diffserv actions on network packets.";
     reference
       "RFC XXXX: A YANG Data Model for Quality of Service (QoS).";
   }
 
   identity default-policy-type {
     base qos-types:policy-type;
     description
-      "This defines default policy-type";
+      "This defines default policy-type.";
   }
 
   identity qos-group {
     base qos-types:filter-type;
     description
-      "qos-group filter-type";
-  }
-
-  identity drop {
-    base qos-types:action-type;
-    description
-      "drop action-type";
+      "Filter-type to match on  packet metadata. This metadata
+       can be used for packet marking, queuing for traffic in
+       egress direction.";
   }
 
   grouping qos-group {
     list qos-group {
-      key "qos-group-min qos-group-max";
+      key "min max";
       description
-        "list of dscp ranges";
-      leaf qos-group-min {
+        "List of qos-group ranges.";
```

Comment:
Something like ..

"List of qos-groups. A QoS group is a set of packet tagged as group."